lib/cloudregister/registerutils.py

```diff
@@ -1888,6 +1897,16 @@ def is_new_registration():
     )
 
 
+# ----------------------------------------------------------------------------
+def is_registration_completed():
+    """Indicate whether a registration is in process or completed based on the
+       marker file. Note it is the responsibility of the process to properly
+       manage the marker file"""
+    return os.path.exists(
```

Comment:
nitpick, could be changed to `os.path.isfile` because that's what we expect

Reply:
fixed in [0d35dac](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/pull/251/commits/0d35dacf5fc286bb2b5d4eaf848bd40e85998357)